src/cloud_provider/service.rs

```diff
@@ -333,8 +333,8 @@ where
         service.engine_error(
             EngineErrorCause::User(
                 "Your application has failed to start. \
-                Ensure you can run it without issues with `qovery run` and check its logs from the web interface or the CLI with `qovery log`",
-            ),
+                Ensure you can run it without issues with `qovery run` and check its logs from the web interface or the CLI with `qovery log`.\
+                HINT: this issue often occurs due to ports misconfiguration. Make sure you exposed the correct port (using EXPOSE statement in Dockerfile or via Qovery configuration). ",
```

Comment:
nice one @pjeziorowski 👍🏽

Reply:
This message is already the HINT message. Please don't provide a HINT into a HINT, I propose you to rephrase it to make it more precise and concise.
Also, it can't compile actually